fe/fe-core/src/main/java/com/starrocks/external/iceberg/IcebergUtil.java

```diff
@@ -0,0 +1,63 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021 StarRocks Limited.
+
+package com.starrocks.external.iceberg;
+
+import com.starrocks.catalog.IcebergTable;
+import com.starrocks.external.hive.HdfsFileFormat;
+import org.apache.iceberg.FileFormat;
+import org.apache.iceberg.catalog.TableIdentifier;
+
+public class IcebergUtil {
+    /**
+     * Get Iceberg table identifier by table property
+     */
+    public static TableIdentifier getIcebergTableIdentifier(IcebergTable table) {
+        return TableIdentifier.of(table.getDb(), table.getTable());
+    }
+
+    /**
+     * Get Iceberg table identifier by table property
+     */
+    public static TableIdentifier getIcebergTableIdentifier(String db, String table) {
+        return TableIdentifier.of(db, table);
+    }
+
+    /**
+     * Returns the corresponding catalog implementation.
+     */
+    public static IcebergCatalog getIcebergCatalog(IcebergTable table)
+            throws StarRocksIcebergException {
+        IcebergCatalogType catalogType = table.getCatalogType();
+        return getIcebergCatalog(catalogType, table.getIcebergHiveMetastoreUris());
+    }
+
+    /**
+     * Returns the corresponding catalog implementation.
+     */
+    public static IcebergCatalog getIcebergCatalog(IcebergCatalogType catalogType, String metastoreUris)
+            throws StarRocksIcebergException {
+        switch (catalogType) {
+            case HIVE_CATALOG: return IcebergHiveCatalog.getInstance(metastoreUris);
```

Comment:
There may be a line break here.

Reply:
since we just return for this case , so we do not need a break, it will cause a compiler error